src/current/_data/releases.yml

```diff
@@ -5762,3 +5762,37 @@
     docker_arm_limited_access: false
   source: true
   previous_release: v24.1.0-alpha.4
+
+- release_name: v23.2.4
+  major_version: v23.2
+  release_date: '2024-04-08'
```

Comment:
Just confirming this is the correct date?

Reply:
good catch! updated to 04-09